airflow/configuration.py

```diff
@@ -378,6 +379,27 @@ def validate(self):
         self._upgrade_postgres_metastore_conn()
         self.is_validated = True
 
+    def _validate_max_tis_per_query(self) -> None:
+        """
+        Check if config ``scheduler.max_tis_per_query`` is not greater than ``core.parallelism``.
+        If not met, a warning message printed, to guide user for best practice.
+
+        More info: https://github.com/apache/airflow/pull/29602
+
+        """
+        # TODO: this type of checking can be generalized like the deprecate checking
```

Comment:
Do we want to do this in this PR? Maybe cut an issue for the ToDo if not (and link it here as well). ToDos in code rarely get done or noticed ever again.

Reply:
yeah, also I'm not really sure how and for what this would be generalized

Reply:
If it's not abundantly clear now then it surely won't be to any future devs. So perhaps just drop it.